OracleGoldenGate/23/README.md

```diff
@@ -0,0 +1,183 @@
+```
+
+### SSL Certificate
+
+When bringing your own SSL certificate to an Oracle GoldenGate container, two files are needed:
+
+1. `ogg.key` - The private key for the SSL certificate.
+1. `ogg.pem` - The SSL leaf certificate, and a full certificate trust chain
+
+If these files are located in a directory called `cert`, they can be used in the GoldenGate container with a volume mount as shown here:
+
+```sh
+$ docker run -v ${PWD}/cert:/etc/nginx/cert:ro -p 8443:443 oracle/goldengate:23.4
+...
+```
+
+The certificate file, `ogg.pem`, must contain a full certificate chain starting with the leaf certificate, and followed by all other certificates in the Certificate Authority chain.
+
+```pem
+-----BEGIN CERTIFICATE-----
+MIIFBTCCA+2gAwIBAgISBJSzNXE+Ha5eDw76N5lgHhTpMA0GCSqGSIb3DQEBCwUA
+MEoxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MSMwIQYDVQQD
+...
+dr7wTE+AQwcOLAGjIvFOL7GK8JrhKvuFvnSoys/1O2CK3vVhBgS+mEF6D+QjIGTv
+VC01LCPT51q58INy4RtDBPSqlJwrzz+pOOWd5rBWhu2UPktVHz3AtYE=
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIEkjCCA3qgAwIBAgIQCgFBQgAAAVOFc2oLheynCDANBgkqhkiG9w0BAQsFADA/
+MSQwIgYDVQQKExtEaWdpdGFsIFNpZ25hdHVyZSBUcnVzdCBDby4xFzAVBgNVBAMT
+...
+PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
+KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
+-----END CERTIFICATE-----
+```
+
+### Running the Administration Client
+
+The **Administration Client** utility can be run with this command:
+
+```sh
+$ docker exec -ti --user ogg <container name> adminclient
+Oracle GoldenGate Administration Client for Oracle
+Version 23.4 ...
+```
+
+### Running Scripts Before Setup and on Startup
+
+The container images can be configured to run scripts before setup and on startup. Currently, `.sh` extensions are supported. For setup scripts just mount the volume `/u01/ogg/scripts/setup` or extend the image to include scripts in this directory. For startup scripts just mount the volume `/u01/ogg/scripts/startup` or extend the image to include scripts in this directory. Both of those locations
+are static and the content is controlled by the volume mount.
+
+The example below mounts the local directory `${PWD}/myScripts` to `/u01/ogg/scripts` which is then searched for custom startup scripts:
+
+```sh
+docker run -v "${PWD}/myScripts:/u01/ogg/scripts" oracle/goldengate:23.4
+```
+
+## Known Issues
+
+None
+
+## License
+
+All scripts and files hosted in this project and GitHub [docker-images/OracleGoldenGate](../) repository required to build the container images are, unless otherwise noted, released under the Universal Permissive License (UPL), Version 1.0.  See [LICENSE](/LICENSE) for details.
+
+To download and run Oracle GoldenGate, regardless of whether inside or outside a container, you must download the binaries from the [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/goldengate/downloads/index.html) and accept the license indicated on that page.
```

Comment:
The URL should use `https` by default. There is a redirect in place but it would be better not to rely on it.